lib/chore/new_relic.rb

```diff
@@ -83,10 +83,10 @@
           # Relevant rpm code:
           # https://github.com/newrelic/rpm/blob/3.15.0.314/lib/new_relic/agent/transaction.rb#L146
```

Comment:
We should probably update the links in the comments to one that's more contemporaneous with the commit date